pkg/controller/servicebindingrequest/binder.go

```diff
@@ -377,10 +376,15 @@ func (b *binder) updateContainer(container interface{}) (map[string]interface{},
 	// effectively binding the application with intermediary secret
 	c.EnvFrom = b.appendEnvFrom(c.EnvFrom, b.sbr.GetName())
 
+	secretRes := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "secrets"}
```

Comment:
Another option here is to modify `newBinder` to accept the `secret` created in `newServiceBinder`, and use `secret.get()`.

Reply:
Yeah, It can be a possibility but adding that here seems to be an overkill IMO